internal/pkg/docker/dockercompose/svc.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockercompose
+
+import (
+	"fmt"
+	"github.com/aws/copilot-cli/internal/pkg/manifest"
+	"github.com/compose-spec/compose-go/types"
+	"time"
+)
+
+func convertBackendService(service *types.ServiceConfig, port uint16) (*manifest.BackendServiceConfig, IgnoredKeys, error) {
```

Comment:
Does the `compose-go` library handle `port` separately? Where is port coming from?

Reply:
There is some complex logic that will be needed to derive the proper value for the `port` that will happen in Milestone 3 - since our model of a service having a single port doesn't exactly line up with how Compose works with its multiple ports. It is being passed in here since that logic will also decide whether the service becomes a Backend Service or a Load Balanced Web Service, so it needs to be called outside of `convertBackendService` anyways.